### PR TITLE
rdatasets: Rename Rdatasets to Rdataset

### PR DESCRIPTION
**Pull request by heronshoes**
This pull request is about the class name in rdatasets.

Class `Rdatasets` represents only one dataset of R datasets. So singular name `Rdataset` may be better.
This request will create an alias and preserve the old one for compatibility.

(This is from the previous comment by @kou in gitter 😄 )

### CODE REVIEW

**Comment by heronshoes** [comment]
OK! Iwill try 👍

**Comment by heronshoes** [comment]
Done !

I did not change `@metadata.id = "rdatasets*` because it is used in the cache name.

File: lib/datasets/rdataset.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ class RdatasetsList < Dataset
     def initialize
       super
       @metadata.id = "rdatasets"
```
Comment (kou) [inline lib/datasets/rdataset.rb:21]:
```suggestion
      @metadata.id = "rdataset-list"
```

File: lib/datasets/rdataset.rb [hunk 2]
```diff
@@ -66,7 +69,7 @@ def initialize(package_name, dataset_name)
 
       super()
       @metadata.id = "rdatasets-#{package_name}-#{dataset_name}"
```
Comment (kou) [inline lib/datasets/rdataset.rb:71]:
```suggestion
      @metadata.id = "rdataset-#{package_name}-#{dataset_name}"
```

**Review by kou — COMMENTED** [review]
Could you also update `@metadata.id`?
We don't need to care about cache too much.

**Comment by heronshoes** [comment]
I have 818MB rdataset cache now (this is for all dataset including .arrow). This change requires manual clean up for older cache.

But it should change 😃

**Comment by kou** [comment]
Hmm. It seems that we need `Datasets.clear_cache!` or something for clearing all caches for Red Datasets...

**Comment by heronshoes** [comment]
I'ts OK. Thanks !
If someone want to clear the cache, they will invoke `rm -r ~/.cache/red-datasets/rdatasets*` or something...

I was worried about hidden cache that users don't notice,
but I think there are only a limited number of people having large amount of cache like me 🐸

**Pull request merged**